src/utils/Processing.ts

```diff
@@ -168,6 +170,70 @@ export function writeFHIRResources(outDir: string, outPackage: Package, snapshot
   logger.info(`Exported ${count} FHIR resources as JSON.`);
 }
 
+/**
+ * Initializes an empty sample FSH within a user specified subdirectory of the current working directory
+ */
+export function init(): void {
+  logger.info(
+    'This tool will initialize a SUSHI project by creating all necessary files to build a simple IG. ' +
+      'The content of this IG will be simple default values, and should be updated. ' +
+      'The tool will now interactively create a config.yaml file for your project.'
+  );
+
+  const configDoc = YAML.parseDocument(
+    fs.readFileSync(path.join(__dirname, 'init-project', 'config.yaml'), 'utf-8')
+  );
+  // Accept user input for certain fields
+  ['name', 'id', 'canonical', 'status', 'version'].forEach(field => {
+    const userValue = readlineSync.question(
+      `${field.charAt(0).toUpperCase() + field.slice(1)} (Default: ${configDoc.get(field)}): `
+    );
+    if (userValue) {
+      configDoc.set(field, userValue);
+    }
+  });
+  // Ensure copyrightYear is accurate
+  configDoc.set('copyrightYear', `${new Date().getFullYear()}+`);
+
+  // Write init directory out, including user made config.yaml, files in utils/init-project, and build scripts from ig/files
+  const outputDir = path.resolve('.', configDoc.get('name'));
+  const initProjectDir = path.join(__dirname, 'init-project');
+  if (!readlineSync.keyInYN(`Initialize SUSHI project in ${outputDir}?`)) {
+    logger.info('Aborting initialization.');
+    return;
+  }
+
+  logger.info(`Adding files to ${outputDir}`);
+  // Add index.md content, updating to reflect the user given name
+  const indexPageContent = fs
+    .readFileSync(path.join(initProjectDir, 'index.md'), 'utf-8')
+    .replace('ExampleIG', configDoc.get('name'));
+  fs.ensureDirSync(path.join(outputDir, 'fsh', 'ig-data', 'input', 'pagecontent'));
+  fs.writeFileSync(
+    path.join(outputDir, 'fsh', 'ig-data', 'input', 'pagecontent', 'index.md'),
+    indexPageContent
+  );
+  // Add the config
+  fs.writeFileSync(path.join(outputDir, 'fsh', 'config.yaml'), configDoc.toString());
+  // Copy over remaining static files
+  fs.copyFileSync(
+    path.join(initProjectDir, 'patient.fsh'),
+    path.join(outputDir, 'fsh', 'patient.fsh')
+  );
+  fs.copyFileSync(
+    path.join(initProjectDir, 'init-gitignore.txt'),
+    path.join(outputDir, '.gitignore')
+  );
+  // Add the _updatePublisher, _genonce, and _gencontinuous scripts
+  const scriptsDir = path.join(__dirname, '..', 'ig', 'files');
+  const scriptsDirContents = fs.readdirSync(scriptsDir);
+  scriptsDirContents
+    .filter(file => file.startsWith('_'))
+    .forEach(file => {
+      fs.copyFileSync(path.join(scriptsDir, file), path.join(outputDir, file));
+    });
```

Comment:
I think my affinity for boxes may be a sickness:
```
╭──────────────────────────────────────────────────────────╮
│          Project initialized at: ./MyProject.            │
├──────────────────────────────────────────────────────────┤
│ Now try this:                                            │
│                                                          │
│ > cd MyProject                                           │
│ > sushi .                                                │
│                                                          │
│ For guidance on project structure and configuration see  │
│ the SUSHI documentation:  https://fshschool.org/sushi    │
╰──────────────────────────────────────────────────────────╯
```

Reply:
I'm on it boxman.